### PR TITLE
Added temporary hacktoberfest animated banner

### PR DESCRIPTION
**Pull request by CodingSpecies**
I added a temporary typing effect banner, about hacktoberfest:

[Typing SVG](https://git.io/typing-svg)

thanks.

### CODE REVIEW

**Review by schmelto — COMMENTED** [review]
For me the typing gif is not fully loading it stops at Contribute there is missing the `NOW` as seen in the git diff.
Mabybe you can take a look at this :)

**Comment by CodingSpecies** [comment]
I will try to fix it. Thanks @schmelto

**Comment by CodingSpecies** [comment]
I tried to fix it @schmelto

File: profile/README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+[![Typing SVG](https://readme-typing-svg.herokuapp.com?vCenter=true&multiline=true&width=500&height=200&lines=%F0%9F%8E%8A+Hacktoberfest%2C+at+EddieHub!;Checkout+Our+Repos!%F0%9F%8E%8A)](https://git.io/typing-svg)
```
Comment (schmelto) [inline profile/README.md:1]:
```suggestion
[![Typing SVG](https://readme-typing-svg.herokuapp.com?vCenter=true&multiline=true&width=500&height=200&lines=%F0%9F%8E%8A+Hacktoberfest%2C+at+EddieHub!;Checkout+Our+Repos!%F0%9F%8E%8A)](https://hacktoberfest.digitalocean.com/)
```

**Review by schmelto — CHANGES_REQUESTED** [review]

File: profile/README.md (hunk 1 above)
Comment (schmelto) [inline profile/README.md:1]:
```suggestion
<a href="https://hacktoberfest.digitalocean.com/"><img src="https://readme-typing-svg.herokuapp.com?vCenter=true&multiline=true&width=500&height=200&lines=%F0%9F%8E%8A+Hacktoberfest%2C+at+EddieHub!;Checkout+Our+Repos!%F0%9F%8E%8A" /></a>
```

**Comment by CodingSpecies** [comment]
@schmelto is this pr part of hacktoberfest? Thanks.

**Pull request merged**